.travis.yml

```diff
@@ -1,2 +1,9 @@
 language: node_js
 node_js: 6
+before_script:
+  - pushd node_modules
+  - git clone https://github.com/TheSavior/babel-plugin-rewire.git
+  - cd babel-plugin-rewire
+  - git checkout ignored-identifiers
+  - npm install
```

Comment:
I don't have to run the build because `babel-plugin-rewire` has a `prepublish` script which runs it (and [`prepublish` runs on install](https://www.youtube.com/watch?v=q2VmFz9VUJ0))